packages/core/src/components/Dropdown/Dropdown.tsx

```diff
@@ -136,6 +136,17 @@ const Dropdown: VibeComponent<DropdownComponentProps, HTMLElement> & {
       return null;
     };
 
+    // SSR support
+    const [WindowedMenuList, setWindowedMenuList] = useState(null);
+    useEffect(() => {
+      if (isClient()) {
+        // Dynamically import the specific named export from react-windowed-select for SSR support
+        import("react-windowed-select").then(module => {
+          setWindowedMenuList(() => module.WindowedMenuList);
+        });
+      }
+    }, []);
```

Comment:
Doesn't it force to import the Dropdown with Suspense?

Reply:
It will re-render after the use effect finish